internal/services/servicebus/servicebus_subscription_resource.go

```diff
@@ -111,6 +111,11 @@ func resourceServiceBusSubscription() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"duplicate_detection_history_time_window": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+			},
```

Comment:
we should validate this value - there is a an existing duration validation function

Reply:
I was trying to search for something like this. I'll have another look at this.